polyfills/Object/defineProperties/polyfill.js

```diff
@@ -1,6 +1,8 @@
 Object.defineProperties = function defineProperties(object, descriptors) {
 	for (var property in descriptors) {
-		Object.defineProperty(object, property, descriptors[property]);
+		if (Object.hasOwnProperty(property)) {
```

Comment:
Are you sure about this?  Seems like if anything it should be checking the inverse of this condition.